### PR TITLE
Api Documentation

### PR DESCRIPTION
**Pull request by CommanderStorm**
As noted in https://github.com/TUM-Dev/navigatum/issues/173 our Documentation is somethimes kind of lacking.

This PR adresses this issue:

Currently, the darkmode is.. 
rough..

resolves https://github.com/TUM-Dev/navigatum/issues/173

### CODE REVIEW

**Comment by CommanderStorm** [comment]
https://github.com/Amoenus/SwaggerDark/ seems like a good solution for the dark mode issue.
What is the best solution, to only include this into the dark mode of this page in your opinion?
All solutions I can think of are in some way bad…

**Comment by octycs** [comment]
What do you think about this approach in https://github.com/TUM-Dev/navigatum/pull/185/commits/9235282a7e2149fdaaf24d43ce5136040876aa69?

File: webclient/package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "description": "",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "install": "napa Amoenus/SwaggerDark#b6fa2536da4e60b3fa15fa5e62eaac668dbc2a21:SwaggerDark"
```
Comment (CommanderStorm) [inline webclient/package.json:7]: Is this necessary?
Wouldn't it be easier to just privately fork `Amoenus/SwaggerDark`, add a `package.json` and handle it like `github:Valexr/spectre`?

Comment (CommanderStorm) [reply webclient/package.json:7]: Doing so could also solve the issue, that [this css file](https://github.com/Amoenus/SwaggerDark/blob/b6fa2536da4e60b3fa15fa5e62eaac668dbc2a21/SwaggerDark.css#L1) overrides some values, we ceartainly dont want overridden..

Comment (CommanderStorm) [reply webclient/package.json:7]: image

File: webclient/gulpfile.js [hunk 2]
```diff
@@ -638,6 +640,26 @@ function copyMapJS() {
 }
 gulp.task("map", gulp.parallel(copyMapCSS, copyMapJS));
 
+// --- api-visualiser (currently swagger-ui) Pipeline ---
+function copyApiCSS() {
+  return gulp
+    .src("node_modules/SwaggerDark/SwaggerDark.css")
+    .pipe(postcss([
+      postcssPrependSelector({ selector: "body.theme-dark " })
```
Comment (CommanderStorm) [inline webclient/gulpfile.js:648]:
```suggestion
      postcssPrependSelector({ selector: "body.theme-dark #swagger-ui " })
```
Alternatively, this works:
image
image

Comment (octycs) [reply webclient/gulpfile.js:648]: This is good in any case I think

File: webclient/package.json (hunk 1 above)
Comment (octycs) [reply webclient/package.json:7]: I also thought about directly including the css file in the repo, but the fork approach is probably a bit more consistent with the rest of the build.

Comment (octycs) [reply webclient/package.json]: I've created a private fork at https://github.com/octycs/SwaggerDark to not clutter the TUM-Dev namespace

**Review by octycs — APPROVED** [review]

**Pull request merged**